bin/logger

```diff
@@ -64,24 +64,69 @@ we redirect the aggregated output back to STDOUT (FD 1). You can also log the
 aggregated output to a single file.
 """
 
+import itertools
```

Comment:
This is just a copy-paste of the entire new version of the `bin/logger` file from h and lms